devTools/phpstan-tests.neon

```diff
@@ -10,6 +10,7 @@ parameters:
     ignoreErrors:
         - '#Return type \(void\) of method .*(Visitor|Traverser).* should be compatible with return type .* of method PhpParser\\Node.*#'
         - '#^Cannot call method (willReturn|shouldBeCalledTimes)\(\) on iterable<Infection\\Mutation\\Mutation>\.$#'
+        - '#Offset .*\\.* does not exist on array<class-string<.*>.*#'
```

Comment:
This is a bogus warning because `array<class-string<SomeInterface>>` cannot contain interfaces, only concrete classes.